src/k8s-vault.cr

```diff
@@ -183,14 +184,15 @@ module K8sVault
   # Return `nil`
   def self.cleanup : Nil
     K8sVault::Log.debug "cleaning up"
-    File.delete(K8sVault::KUBECONFIG_TEMP) rescue nil
+    File.delete(ENV["KUBECONFIG"]) rescue nil
   end
 
   # Runs everything
   def self.run(options : Array(String))
     kubecontext = "_unset_"
     spawn_shell = false
-
+    kubeconfigTemp = K8sVault::KUBECONFIG_TEMP;
```

Comment:
Let's store it in `config. kubeconfigTemp` (record Config) instead 🙏 and use `config. kubeconfigTemp` in place of `K8sVault::KUBECONFIG_TEMP` in following code.
Deleting `ENV["KUBECONFIG"]` makes me uneasy 😅 especially, since the `self.cleanup` will wire before we pass the new `KUBECONFIG` to the `Process.run` on line 341

Reply:
Yes, I was not very that use ENV var was a good idea.  
But, how can I get the config value in the cleanup method ? It's the first time that I develop with Casper lang. :relaxed:

Reply:
Could you advise me on retrieve config in the cleanup function ?

Reply:
@anapsix I tried to don't use `ENV["KUBECONFIG"]`. I chose to pass a param to cleanup function because `config` var might not be set.  
Do you think this is correct ?

Reply:
Sorry Louis, kinda busy with work. Will poke at it next week.

Reply:
kinda swamped at work at the moment, if your solution works for you, then that should hopefully serve the purpose while I get around to it